Project3/index11.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">  
+    <title>4:00 P.M.</title>
+    <div class="card">
+    <h1>4:00 P.M.</h1>
+    <h2>Class Starts</h2></head>
+<body style="background-color:#000000"; >
+    <p>I made my bed, now I gotta lie in it.<br />
+        Actually, I crapped in the bed, now I gotta lie on it.
+    </p>
+    
```

Comment:
At the point this project was due, this and five other pages were missing their photos.